studio/components/interfaces/Home/NewProjectPanel/APIKeys.tsx

```diff
@@ -121,15 +119,15 @@ const APIKeys = () => {
                 </div>
               }
               copy={canReadAPIKeys && isNotUpdatingJwtSecret}
-              reveal={anonKey.tags !== 'anon' && canReadAPIKeys && isNotUpdatingJwtSecret}
+              reveal={anonKey?.tags !== 'anon' && canReadAPIKeys && isNotUpdatingJwtSecret}
```

Comment:
@joshenlim I'm a bit confused by the logic here. On line 48, we find the first key where `tags === 'anon'` so this negation is true only if no such key is found? Why do we set it to reveal if there is no anon key?

Also line 112 wants to split up `tags` by comma but we already filtered by tags on line 48.